kfs-prje/src/main/java/edu/arizona/kfs/module/prje/businessobject/PRJEAuditItem.java

```diff
@@ -0,0 +1,229 @@
+package edu.arizona.kfs.module.prje.businessobject;
+
+import java.util.LinkedHashMap;
+
+import org.kuali.kfs.gl.businessobject.OriginEntryFull;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.bo.PersistableBusinessObjectBase;
+
+import edu.arizona.kfs.module.prje.PRJEPropertyConstants;
+import edu.arizona.kfs.module.prje.dataaccess.PRJETransferRecord;
```

Comment:
I didn't notice it was in the wrong package when I took over the mod. Moving to businessobject package